### ISSUE TITLE
Windows: Pre-built SDK configuration engine cannot be built using Visual Studio 2019

### ISSUE THREAD

**Issue by LB-ArturZieba** [o3de/o3de#16015]
**Describe the bug**
Pre-built SDK configuration fails to finish the build when ran using Visual Studio 2019 with error:
```
C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\VC\Tools\MSVC\14.29.30133\include\xlocinfo.h(18,17): error C5032: detected #pragma warning(push) with no corresponding #pragma warning(pop) [C:
\o3de\build\windows_vs2019\Code\Tools\TestImpactFramework\Runtime\Python\Code\TestImpact.Runtime.Python.Static.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\VC\Tools\MSVC\14.29.30133\include\xlocinfo.h(18,17): error C5032: #pragma warning(push, _STL_WARNING_LEVEL) [C:\o3de\build\windows_vs2019\Code\
Tools\TestImpactFramework\Runtime\Python\Code\TestImpact.Runtime.Python.Static.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\VC\Tools\MSVC\14.29.30133\include\xlocinfo.h(18,17): error C5032:                 ^ [C:\o3de\build\windows_vs2019\Code\Tools\TestImpactFramewor
k\Runtime\Python\Code\TestImpact.Runtime.Python.Static.vcxproj]
```
For detailed log please refer to the Pre-builtSDKConfigBuildLog.txt file attached in the **Command Prompt Log** section.

It should be noted that this issue does not occur on Source builds or on Pre-built SDK builds performed using Visual Studio 2022.

Visual Studio 2019 versions used on different machines to reproduce this issue were 16.11.26 and 16.11.8.

**Prerequisites**
* Visual Studio 16.11.x installed (https://www.o3de.org/docs/welcome-guide/requirements/#microsoft-visual-studio)

**Steps to reproduce**
1. Clone the o3de/o3de repository.
2. Create o3de-packages folder in the C:\ path (`mkdir C:\o3de-packages` in Command Prompt).
3. Run the python script from the o3de folder (`python\get_python.bat` in Command Prompt).
4. Configure the build using Visual Studio 16 (`cmake -B build/windows_vs2019 -G "Visual Studio 16" -DLY_3RDPARTY_PATH=C:\o3de-packages -DLY_VERSION_ENGINE_NAME=o3de-install -DCMAKE_INSTALL_PREFIX=C:\o3de-install` in Command Prompt).
5. Build the engine (`cmake --build build/windows_vs2019 --target INSTALL --config profile -- -m` in Command Prompt).

**Expected behavior**
Build finishes, C:\o3de-install folder is generated with Pre-built SDK build of the engine.

**Actual behavior**
Build does not finish and the C:\o3de-install folder is not generated.

**Command Prompt Log**

[Pre-builtSDKConfigBuildLog.txt](https://github.com/o3de/o3de/files/11562030/Pre-builtSDKConfigBuildLog.txt)

**Found in Branch**
Development (0b04b76)

**Commit ID from [o3de/o3de](https://github.com/o3de/o3de) Repository**
0b04b763f011dc87e07bba9e977183c364c2c25a

**Desktop**
 - Device: PC
 - OS: Windows
 - Version 10
 - CPU AMD Ryzen 5 3600X 6-Core Processor
 - GPU AMD Radeon RX 5700 XT
 - Memory 16GB

**Comment by lumberyard-employee-dm** [comment]
This was fixed @mbalfour-amzn in PR https://github.com/o3de/o3de/pull/16023

**Comment by LB-ArturZieba** [comment]
Issue verified as fixed on the Development (9b07c7c) build.
